apps/fileworker/src/routes/front/upload/+page.server.ts

```diff
@@ -0,0 +1,15 @@
+export const actions = {
+	upload: async ({ request }) => {
```

Comment:
Why not do `await fetch('/api/file/:id/:filename` from `page.svelte`? See here:
https://github.com/jahands/fileworker/blob/main/apps/fileworker/src/routes/front/deletion/%2Bpage.svelte#L34